.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: Test
+on: [pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read .nvmrc
+      run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+      id: nvm
+
+    - name: Install Ubuntu packages
+      run: sudo apt-get install libxml2-dev libxslt1-dev python-dev
+
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '${{ steps.nvm.outputs.NVMRC }}'
+
+    - uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+
+    - name: Setup python (${{ matrix.python-version }})
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Setup Python cache
+      uses: actions/cache@v2
+      with:
+        path: venv
+        key: venv-${{ runner.os }}-${{ steps.full-python-version.outputs.version }}-${{ hashFiles('**/requirements-dev.txt') }}
+
+    - name: Run python tests
+      env: 
+        PYTEST_ARGS: --cov=app --cov-report=term-missing
```

Comment:
If we're not using coveralls, do we need this?

Reply:
I wasn't sure if some other tooling used this but if not, I'm happy to remove both this and `coverage`

Reply:
removed in 919a4c3